test/relations.test.js

```diff
@@ -4987,7 +4987,7 @@ describe('relations', function() {
       Category.findById(category.id, function(err, cat) {
         if (err) return done(err);
         var link = cat.items.at(0);
-        link.updateAttributes({notes: 'Updated notes...'}, function(err, link) {
+        cat.items.updateById(link.id, {notes: 'Updated notes...'}, function(err, link) {
```

Comment:
`link.destroy` still works in https://github.com/strongloop/loopback-datasource-juggler/blob/124b1d48839830f1f69b68fb90217790700b3ae6/test/relations.test.js#L5012 , I am looking into why `link.updateAttributes` returns error with postgresql